custom_components/fraimic/image_convert.py

```
@@ -209,6 +229,139 @@ def _preprocess(
     return image
 
 
+def _official_prepare_image(
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_official_prepare_image`

(ANN202)